inputFiles/solidMechanics/triaixalDriverTables/axialStrain.geos

```diff
@@ -0,0 +1,6 @@
+0.0

```

Comment:
i would actually put these files in something like`inputFiles/triaxialDriver/tables`.  We organize mainly by physics solver, and the driver is unrelated to the LagrangianFEM examples already in `solidMechanics`.  Also, note you have a typo in the path: `triaixial`